lms/envs/common.py

```diff
@@ -535,11 +535,6 @@
 STUDENT_FILEUPLOAD_MAX_SIZE = 4 * 1000 * 1000  # 4 MB
 MAX_FILEUPLOADS_PER_INPUT = 20
 
-# FIXME:
-# We should have separate S3 staged URLs in case we need to make changes to
-# these assets and test them.
```

Comment:
@pmitros FYI, you wrote this comment in 2/20/2012. Just flagging you in case you know a reason why LIB_URL shouldn't be deleted.